### ISSUE TITLE
Checked-In `examples/ctapcli/target` Directory Makes Repo Quite Large

### ISSUE THREAD

**Issue by gkelly** [gebogebogebo/ctap-hid-fido2#24]
There's a `target` directory checked in under `examples/ctapcli/target`. This makes a checkout ~228MB. You might want to consider rewriting history to remove that directory, as just `git rm`'ing it will leave the objects reachable from history.

**Comment by gebogebogebo** [comment]
@gkelly 
Thanks for the advice!
Removed `examples/ctapcli/target`!

**Comment by gebogebogebo** [comment]
I need to delete it completely from my history...

**Comment by gkelly** [comment]
Yes, sorry. That's what I meant by rewrite history. There's a GitHub article that might be helpful: https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/removing-sensitive-data-from-a-repository#purging-a-file-from-your-repositorys-history. It's aimed at sensitive data but the process is the same.

**Comment by gebogebogebo** [comment]
Deleted `examples/ctapcli/target`
This reduced the size to about 130M when I checked out.
:smile:

**Comment by gebogebogebo** [comment]
@gkelly Thanks for the advice!
:laughing:
